calculator/operations.py

```diff
@@ -1,20 +1,29 @@
 
 def add(*args):
-    # your implementation here
-    pass
+    x = float(sum(args))
+    return x
 
 def subtract(*args):
     # your implementation here
-    pass
-
+    x = float(args[0])
+    for arg in args[1:]:
+        x = x-float(arg)
+    return x
+    
```

Comment:
Could use reduce functions on most of these for less lines but you'd also have to account for using Python 3.